docker-hub/api/latest.yaml

```diff
@@ -875,6 +880,267 @@ paths:
+          description: ""
+          example: 1
+        - name: count
+          in: query
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 200
+          description: ""
+          example: 10
+        - name: filter
+          in: query
+          schema:
+            type: string
+          description: ""
+          example: userName eq "jon.snow@docker.com"
+        - $ref: "#/components/parameters/scim_attributes"
+        - name: sortOrder
+          in: query
+          schema:
+            type: string
+            enum:
+              - ascending
+              - descending
+        - name: sortBy
+          in: query
+          schema:
+            type: string
+          description: "User attribute to sort by."
+          example: userName
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_users_resp"
+        400:
+          $ref: "#/components/responses/scim_bad_request"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        403:
+          $ref: "#/components/responses/scim_forbidden"
+        404:
+          $ref: "#/components/responses/scim_not_found"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+    post:
+      summary: Create user
+      description: |
+        Creates a user. If the user already exists by email, they are assigned
+        to the organization on the "company" team.
+      tags: [ scim ]
+      security:
+        - bearerSCIMAuth: [ ]
+      requestBody:
+        $ref: "#/components/requestBodies/scim_create_user_request"
+      responses:
+        201:
+          $ref: "#/components/responses/scim_create_user_resp"
+        400:
+          $ref: "#/components/responses/scim_bad_request"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        403:
+          $ref: "#/components/responses/scim_forbidden"
+        404:
+          $ref: "#/components/responses/scim_not_found"
+        409:
+          $ref: "#/components/responses/scim_conflict"
+        500:
+          $ref: "#/components/responses/scim_error"
+
+  /v2/scim/2.0/Users/{id}:
+    parameters:
+      - $ref: "#/components/parameters/scim_user_id"
+    get:
+      summary: Get a user
+      description: |
+        Returns a user by ID.
+      tags: [ scim ]
+      security:
+        - bearerSCIMAuth: [ ]
+      responses:
+        200:
+          $ref: "#/components/responses/scim_get_user_resp"
+        400:
+          $ref: "#/components/responses/scim_bad_request"
+        401:
+          $ref: "#/components/responses/scim_unauthorized"
+        403:
+          $ref: "#/components/responses/scim_forbidden"
+        404:
+          $ref: "#/components/responses/scim_not_found"
+        500:
+          $ref: "#/components/responses/scim_error"
+    put:
+      summary: Update a user
+      description: |
+        Updates a user. This route is used to change the user's name, activate,
```

Comment:
Updates a user. Use this route to change the user's name, activate,
        and deactivate the user.